routing/router.go

```diff
@@ -2127,6 +2127,11 @@ func (r *ChannelRouter) ForAllOutgoingChannels(cb func(*channeldb.ChannelEdgeInf
 
 	return r.selfNode.ForEachChannel(nil, func(_ *bolt.Tx, c *channeldb.ChannelEdgeInfo,
 		e, _ *channeldb.ChannelEdgePolicy) error {
+		
+		// Skip channels with unknown policy
```

Comment:
But then the function name ForAllOutgoingChannels doesn't cover the logic anymore. Because it will also make callbacks with just a nil value and what can you do with that as a caller?

Reply:
Good point. 

See my other comment below about returning an error if this situation occurs.